pkg/tsdb/postgres/macros.go

```diff
@@ -109,6 +109,9 @@ func (m *postgresMacroEngine) evaluateMacro(name string, args []string) (string,
 		}
 
 		if m.timescaledb {
+			if interval.Seconds() < 1.0 {
```

Comment:
instead of checking it is < 1.0 could we check float(int64(interval.Seconds())) == interval.Seconds(), or something similar, because in this case, we could keep the possibility to have an interval like 1.2 seconds. (in reality it might be less but i think 1.5 is still something that users might use.)

Reply:
Changed so that always a precision of 1 is used.